File: Clase.N2/Programacion 3/src/Clases/Animal.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package Clases;

import java.time.LocalDate;

/**
 *
 * @author LENOVO
 */
public class Animal {
    private String nombre;
    private int peso;
    private String propietario;
    private LocalDate fecha_cumpleaños;
    private LocalDate fecha_ultima_vacuna;
    
    //constructor
    public Animal(String nombre, int peso, String propietario, LocalDate fecha_cumpleaños, LocalDate fecha_ultima_vacuna){
        this.nombre = nombre;
        this.peso = peso;
        this.propietario = propietario;
        this.fecha_cumpleaños = fecha_cumpleaños;
        this.fecha_ultima_vacuna = fecha_ultima_vacuna;
    }

    /**
     * @return the nombre
     */
    public String getNombre() {
        return nombre;
    }

    /**
     * @param nombre the nombre to set
     */
    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    /**
     * @return the peso
     */
    public int getPeso() {
        return peso;
    }

    /**
     * @param peso the peso to set
     */
    public void setPeso(int peso) {
        this.peso = peso;
    }

    /**
     * @return the propietario
     */
    public String getPropietario() {
        return propietario;
    }

    /**
     * @param propietario the propietario to set
     */
    public void setPropietario(String propietario) {
        this.propietario = propietario;
    }

    /**
     * @return the fecha_cumpleaños
     */
    public LocalDate getFecha_cumpleaños() {
        return fecha_cumpleaños;
    }

    /**
     * @param fecha_cumpleaños the fecha_cumpleaños to set
     */
    public void setFecha_cumpleaños(LocalDate fecha_cumpleaños) {
        this.fecha_cumpleaños = fecha_cumpleaños;
    }

    /**
     * @return the fecha_ultima_vacuna
     */
    public LocalDate getFecha_ultima_vacuna() {
        return fecha_ultima_vacuna;
    }

    /**
     * @param fecha_ultima_vacuna the fecha_ultima_vacuna to set
     */
    public void setFecha_ultima_vacuna(LocalDate fecha_ultima_vacuna) {
        this.fecha_ultima_vacuna = fecha_ultima_vacuna;
    }
    
}

File: Python/AT-1/Clases/Animal.py
class Animal:
    def __init__(self, nom, peso, prop, fecha_cump, fecha_ult_v):
        self.__nombre = nom
        self.__peso = peso
        self.__propietario = prop
        self.__fecha_cumpleanios = fecha_cump
        self.__fecha_ultima_vacuna = fecha_ult_v

    # GET
    def get_nombre(self):
        return self.__nombre

    def get_peso(self):
        return self.__peso

    def get_propietario(self):
        return self.__propietario

    def get_fecha_cumpleanios(self):
        return self.__fecha_cumpleanios

    def get_fecha_ultima_vacuna(self):
        return self.__fecha_ultima_vacuna

    # SET
    def set_nombre(self, nom):
        self.__nombre = nom

    def set_peso(self, peso):
        self.__peso = peso

    def set_propietario(self, prop):
        self.__propietario = prop

    def set_fecha_cumpleanios(self, fecha_cump):
        self.__fecha_cumpleanios = fecha_cump

    def set_fecha_ultima_vacuna(self, fecha_ult_v):
        self.__fecha_ultima_vacuna = fecha_ult_v
